WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -15,7 +15,11 @@
         <% elsif competition.in_progress? %>
           <%= icon('fas', 'hourglass-half', title: t('competitions.index.tooltips.hourglass.in_progress'), data: { toggle: "tooltip" }) %>
         <% else %>
-          <%= icon('fas', 'hourglass-start', title: t('competitions.index.tooltips.hourglass.starts_in', days: t('common.days', count:(competition.start_date - Date.today).to_i)), data: { toggle: "tooltip" }) %>
+          <% if @by_announcement_selected %>
+            <%= icon('fas', 'hourglass-start', title: t('competitions.index.tooltips.hourglass.announced_on', announcement_date: competition.announced_at.strftime(" %F %H:%M:%S")), data: { toggle: "tooltip" }) %>
```

Comment:
We actually have a helper to display a localized time, you can see it used [here](https://github.com/thewca/worldcubeassociation.org/blob/d54b152d1dc6c0c0438cabbfef750230e6ced60d/WcaOnRails/app/views/registrations/register.html.erb#L14).
It should be something like:
```suggestion
            <%= icon('fas', 'hourglass-start', title: t('competitions.index.tooltips.hourglass.announced_on', announcement_date: wca_local_time(competition.announced_at)), data: { toggle: "tooltip" }) %>
```

Reply:
wca_local_time outputs HTML which doesn't get parsed inside a tooltip, I've been playing around with other formatting options but I'm leaning on keeping what I already had. Also precision is not that important in this context as most likely people would only be looking at announcements in the last 15 days or so.